src/Target/DrushTargetMetadataTrait.php

```diff
@@ -25,6 +25,17 @@ public function metadataPhpVersion()
       return phpversion();
     });
   }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function metadataThemePath()
+  {
+    $drush = DrushRouter::createFromTarget($this);
+    return $drush->evaluate(function () {
+      return \Drupal::theme()->getActiveTheme();
```

Comment:
The govcms target for codescan and linting is not the active theme, but the custom themes directory. As you can deploy multiple custom themes, base themes, admin theme, etc. #223

Reply:
Is there a way to pull this data from Drupal directly using the `evaluate` function? I.e. Does Drupal know how to generically pull this information? If not, then it may not be the Drush target that should provide this information.

Reply:
I believe that drupal scans for themes recursively in certain locations, and doesn't care about the difference between contrib/custom. So I think it's specific to govcms' requirements to scan this location - effectively making it a "whitelisted location" corresponding to how SaaS still works in govCMS 2.0.

(If I didn't already mention, note that when they run drutiny on a classic SaaS repo (ie it *only* has themes and config) then it fails to get the path, but it still works because it just scans everything from the root.)